### ISSUE TITLE
Source AS leaving and entering an interface with volume

### ISSUE THREAD

**Issue by cardigliano** [ntop/ntopng#3296]
ISPs need to know the source AS leaving an interface and the source AS coming through that interface and volume, it would be nice to have this in the traffic report.

Whatever is leaving the ISP network (egress) should have the source AS number of the ISP only or customers behind the ISP network having an AS number that the ISP acts as transit for them. In the ingress part (whatever is coming through that specific interface), we expect to see diverse AS coming from around the world.
 
In short: source AS and volume ingressing and egressing each physical interfaces is required.
This helps for visibility of what is coming in and out the ISP network and on which pipes it is coming.

<img width="560" alt="Screen Shot 2020-01-23 at 11 28 12" src="https://user-images.githubusercontent.com/7548711/72976949-c1616200-3dd3-11ea-8bea-8d81d7f669f2.png">

**Comment by cardigliano** [comment]
This was a requirement for a specific customer, I guess this is no longer required (we added more information for AS, and more flexibility with traffic disaggregation in the meanwhile). We will reopen this in case it still applies.
